packages/canon/report.api.md

```diff
@@ -0,0 +1,2628 @@
+## API Report File for "@backstage/canon"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+/// <reference types="react" />
+
+import { CSSProperties } from 'react';
+import { JSXElementConstructor } from 'react';
+import { default as React_2 } from 'react';
+import { ReactElement } from 'react';
+import { ReactNode } from 'react';
+import { ResponsiveArray } from '@vanilla-extract/sprinkles';
+
+// @public (undocumented)
+export const Box: ({
+  as,
+  className,
+  style,
+  ...props
+}: BoxProps) => ReactElement<
+  {
+    className: string;
+    style: CSSProperties | undefined;
+  },
+  string | JSXElementConstructor<any>
+>;
+
+// @public
+export type BoxProps = Parameters<typeof boxSprinkles>[0] &
+  Omit<
+    React.AllHTMLAttributes<HTMLElement>,
+    keyof Parameters<typeof boxSprinkles>[0]
+  > & {
+    as?: keyof JSX.IntrinsicElements;
+  };
+
+// @public (undocumented)
+export const boxSprinkles: ((
```

Comment:
Hmm, the size of this thing is a bit of an issue, we probably want to clean up the definition to use reusable types or smth. Bit hard to see what's going on right now 😅. Can leave it for a bit later though.

What is the actual API that this represents though, is it custom style options only available for the box?

Reply:
Yes this type definition is generated by Vanilla extract and we are using it to create all props for the Box component.